airflow/cli/simple_table.py

```diff
@@ -118,8 +118,9 @@ def print_as(
             dict_data = data
         else:
             raise ValueError("To tabulate non-dictionary data you need to provide `mapper` function")
-        dict_data = [{k: self._normalize_data(v, output) for k, v in d.items()} for d in dict_data]
-        renderer(dict_data)
+        dict_data = [{k: self._normalize_data(v, output) for k, v in d.items()} for d in dict_data if d]
+        if dict_data:
+            renderer(dict_data)
```

Comment:
Still needed?

Reply:
I'll remove it from this PR. I'll raise a separate PR because I notice sometime rendering empty list like `[]`